src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -1069,16 +1069,15 @@ test_select_after_try_once (void)
 
 
 static void
-test_multiple_selection_errors (void *context)
+test_multiple_selection_errors (void)
 {
-   const char *uri = "mongodb://doesntexist,example.com:2/?replicaSet=rs"
-                     "&connectTimeoutMS=100";
+   const char *uri =
+      "mongodb://doesntexist.invalid,example.invalid/?replicaSet=rs"
+      "&connectTimeoutMS=100";
```

Comment:
```suggestion
   const char *const uri = "mongodb://doesntexist.invalid,example.invalid/"
                           "?replicaSet=rs&connectTimeoutMS=100";
```
Slightly nicer formatting + const.